tests/test_uart.py

```diff
@@ -93,9 +93,16 @@ def test_rst_frame_received(gw):
 
 
 def test_rstack_frame_received(gw):
-    gw._reset_future = mock.MagicMock()
```

Comment:
It was a left over from, when I tried to clear the reset_future after the result was set.